examples/stubbing-spying__intercept/cypress/integration/intercept-force404.spec.js

```diff
@@ -0,0 +1,29 @@
+/// <reference types="Cypress" />
+
+describe("intercept", () => {
```

Comment:
The `cy.route` `force404` option docs say this:
> Forcibly send a 404 status when the XHR does not match any existing `cy.route()`.

In order for this example to cover the `force404` use case, it should show how to define at least one explicit intercept along with the 404 "default" and then show, in a test, how the defined network request will pass, but an undefined request will error.

Reply:
Done

Reply:
@cowboy if there's nothing else on this one, please resolve it.